sdkproject/Assets/Mapbox/Unity/Editor/Build/Mapbox_iOS_build.cs

```diff
@@ -22,7 +24,15 @@ public static void AppendBuildProperty(BuildTarget buildTarget, string pathToBui
 				proj.ReadFromString(file);
 				string target = proj.TargetGuidByName("Unity-iPhone");
 
-				proj.AddBuildProperty(target, "HEADER_SEARCH_PATHS", "$(SRCROOT)/Libraries/Mapbox/Core/Plugins/iOS/MapboxMobileEvents/include");
+				var defaultIncludePath = "Mapbox/Core/Plugins/iOS/MapboxMobileEvents/include";
+				var includePaths = Directory.GetDirectories(Application.dataPath, "include", SearchOption.AllDirectories);
+				var includePath = includePaths
+					.Select(path => Regex.Replace(path, Application.dataPath + "/", ""))
+					.Where(path => path.EndsWith(defaultIncludePath))
```

Comment:
Can you make this case insensitive and culture invariant, please?

Reply:
Of course, done.